homeassistant/components/srp_energy/config_flow.py

```diff
@@ -0,0 +1,67 @@
+"""Config flow for SRP Energy."""
+import logging
+
+from srpenergy.client import SrpEnergyClient
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_ID, CONF_NAME, CONF_PASSWORD, CONF_USERNAME
+
+from .const import CONF_IS_TOU, DEFAULT_NAME, DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for SRP Energy."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    options = {
```

Comment:
This can be misinterpreted as config entry options which it is not, I think a better choice in naming it would be preferred, like config or something